server/src/main/java/io/spine/server/entity/Migration.java

```diff
@@ -238,6 +239,7 @@ private void warnOnNoSystemEventsPosted() {
         private boolean physicallyRemoveRecord;
 
         private final E entity;
+        private @MonotonicNonNull Transaction<I, E, S, ?> tx;
         private final RecordBasedRepository<I, E, S> repository;
 
         private @MonotonicNonNull Event systemEvent;
```

Comment:
Same for `@LazyInit`. It looks like we should add the annotation to the `systemEvent` as well.